packages/core/src/agent/BaseAgent.ts

```diff
@@ -118,6 +121,7 @@ export abstract class BaseAgent<AgentModules extends ModulesMap = EmptyModuleMap
       this.mediationRecipient,
       this.messagePickup,
       this.basicMessages,
+      this.drpcMessages,
```

Comment:
I'm not sure this protocol should be part of core. Do you think it can be moved to a separate module?